easybuild/easyconfigs/s/Simple-DFTD3/Simple-DFTD3-0.7.0-intel-2022b.eb

```diff
@@ -0,0 +1,50 @@
+# A. Domingo (Vrije Universiteit Brussel)
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'CMakeNinja'
+
+name = 'Simple-DFTD3'
+version = '0.7.0'
+
+homepage = 'https://dftd3.readthedocs.io'
+description = """Reimplementation of the D3 dispersion correction. The s-dftd3 project aims to
+provide a user-friendly and uniform interface to the D3 dispersion model and
+for the calculation of DFT-D3 dispersion corrections."""
+
+toolchain = {'name': 'intel', 'version': '2022b'}
+# In EasyBuild 4.8.2 is a bug in the Intel toolchain, so we need to turn off OpenMP here and add 
+# -fopenmp instead as Bart Oldman suggested
+toolchainopts = {'usempi': False, 'openmp': False, 'pic': True, 'extra_fflags': '-fopenmp'}
+
+github_account = 'dftd3'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['19400a176eb4dcee7b89181a5a5f0033fe6b05c52821e54896a98448761d003a']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+]
+
+dependencies = [
+    ('mctc-lib', '0.3.1'),
+    ('mstore', '0.2.0'),
+    # ('TOML-Fortran', '0.4.1'), # Vesion used in code is 0.2.4
```

Comment:
```suggestion
    ('TOML-Fortran', '0.4.1'),
```

Reply:
That was my initial thought as well, using the version in EasyBuild as well. However, I ran into trouble and looking into the source code in the `subprojects` folder the `toml-f.wrap` reads:
```
[wrap-git]
directory = toml-f
url = https://github.com/toml-f/toml-f
revision = v0.2.4
```
So given the source code is explicitly referring to that particular version, I decided to do it like that.

Furthermore, `xtb` (PR #19225) is using this version:
```
[wrap-git]
directory = toml-f
url = https://github.com/toml-f/toml-f
revision = v0.2.3
```
To add more confusion, `tblite-0.3.0` (PR #19219) uses:
```
directory = toml-f
url = https://github.com/toml-f/toml-f
revision = v0.2.4
```
The current `xtb-6.6.0-foss-2022a.eb` is not using a `simple-DFTD3` module but pulls it down during installation. 
So, with all of that said, and given I had some issues with the test jobs and further up the software chain, I thought it is best to stick to the 0.2.x version and here 0.2.4 was the latest release. 

Any suggestions?

Reply:
so, when I included the `('TOML-Fortran', '0.4.1')` dependency and rebuilt `Simple-DFTD3`, it found the dependency and was happy to use it:

```
-- toml-f: Find installed package
-- Configuring done
-- Generating done
```

and without including it:

```
-- toml-f: Find installed package
-- Could NOT find toml-f (missing: toml-f_DIR)
-- Retrieving toml-f from https://github.com/toml-f/toml-f
```

Reply:
Yep, that is correct here. However, as `Simple-DFT3` is a requirement for `tblite` (PR #19219), when using here the `0.4.1` version of TOML I get in tblite:
```
/dev/shm/easybuild/tblite/0.3.0/foss-2022b-toml-v0.4.1/tblite-0.3.0/src/tblite/param/serde.f90:159:10:

  159 |    ser = toml_serializer(unit)
      |          1
Error: Component output at (1) is a PRIVATE component of toml_serializer
```
So that is the issue here. 
If you like, we can use a `versionsuffix` so both can be used.